mailpoet/changelog.txt

```diff
@@ -1,4 +1,6 @@
 == Changelog ==
+= 5.11.0 - 2025-04-01 =
```

Comment:
Yeah, we should probably establish a rule like "Next". By the time of committing the change we do not know in which version/on which date the change lands.

But this is something we need to document with the linear switch.

Reply:
Updated in https://github.com/mailpoet/mailpoet/pull/6147/commits/59286b8c68f522bf2dd6776bd26a28ca4e7842b3 according to pcNwfB-4Ov-p2